src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Ssl.cs

```diff
@@ -135,7 +136,7 @@ internal static byte[] SslGetAlpnSelected(SafeSslHandle ssl)
 
         internal static string GetOpenSslCipherSuiteName(SafeSslHandle ssl, TlsCipherSuite cipherSuite, out bool isTls12OrLower)
         {
-            string ret = Marshal.PtrToStringAnsi(GetOpenSslCipherSuiteName(ssl, (int)cipherSuite, out int isTls12OrLowerInt));
+            string ret = Marshal.PtrToStringAnsi(GetOpenSslCipherSuiteName(ssl, (int)cipherSuite, out int isTls12OrLowerInt))!;
```

Comment:
Looking at the [C implementation](https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/src/libraries/Native/Unix/System.Security.Cryptography.Native/pal_ssl.c#L383), `CryptoNative_GetOpenSslCipherSuiteName` can return `null`. And the one call site of this method handles getting `null` back. So maybe this method should return `string?`.